test_remember_me.py

```
@@ -0,0 +1,182 @@
+        const data = {};
+        for (let i = 0; i < sessionStorage.length; i++) {
+            const key = sessionStorage.key(i);
+            data[key] = sessionStorage.getItem(key);
+        }
+        return data;
+    }""")
+    
+    required_keys = ["user_email", "remember_me_enabled"]
+    for key in required_keys:
+        if key not in storage:
+            raise RememberMeTestError(f"Missing required session storage key: {key}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)